### PR TITLE
Update directory-country.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Add more details description to the Directory Country GraphQL page.

## Purpose of this pull request
Added Details Description of the Country GraphQL Page in Dev docs. Also added difference between two same query of GraphQL related to the country vs countries query. 

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/graphql/queries/directory-country.html
https://devdocs.magento.com/guides/v2.4/graphql/queries/directory-country.html

### CODE REVIEW

File: src/guides/v2.3/graphql/queries/directory-country.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ redirect_from:
   - /guides/v2.3/graphql/reference/directory.html
 ---
 
-Use the `country` query to retrieve information about a specific country.
+Use the `country` query to retrieve information about a specific country. Dont confuse with [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve list of all the available country in the system.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/directory-country.md:8]:
```suggestion
Use the `country` query to retrieve information about a specific country. Do not confuse with [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve list of all the available countries in the system.
```

File: src/guides/v2.3/graphql/queries/directory-country.md [hunk 2]
```diff
@@ -5,15 +5,15 @@ redirect_from:
...
 
 ## Syntax
 
 `{country(id: String) {Country}}`
 
 ## Example usage
 
-The following query returns information about Australia.
+The following query returns information about Australia. You must have to pass `id` as two-letter abbreviation of the country to retrieve country detalis.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/directory-country.md:16]:
```suggestion
The following query returns information about Australia. You must pass `id` as a two-letter abbreviation of the country to retrieve country details.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please update PR description to the required format and information. A good example : https://github.com/magento/devdocs/pull/8296

**Comment by RakeshJesadiya** [comment]
@BarnyShergold Thank you for the suggestion. I have updated the details.

File: src/guides/v2.3/graphql/queries/directory-country.md [hunk 3]
```diff
@@ -5,15 +5,15 @@ redirect_from:
   - /guides/v2.3/graphql/reference/directory.html
 ---
 
-Use the `country` query to retrieve information about a specific country.
+Use the `country` query to retrieve information about a specific country. Do not confuse with [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve list of all the available countries in the system.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/directory-country.md:8]:
```suggestion
Use the `country` query to retrieve information about a specific country. Do not confuse with the [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve a list of all the available countries in the system.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Apologies - I missed 2 small corrections.
All good after these.

**Comment by RakeshJesadiya** [comment]
@BarnyShergold No Problem, I have corrected that also. Thanks.

**Review by BarnyShergold — APPROVED** [review]
Excellent!

File: src/guides/v2.3/graphql/queries/directory-country.md [hunk 4]
```diff
@@ -5,15 +5,15 @@ redirect_from:
   - /guides/v2.3/graphql/reference/directory.html
 ---
 
-Use the `country` query to retrieve information about a specific country.
+Use the `country` query to retrieve information about a specific country. Do not confuse with the [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query that will be used to retrieve a list of all the available countries in the system.
```
Comment (hguthrie) [inline src/guides/v2.3/graphql/queries/directory-country.md:8]:
```suggestion
Use the `country` query to retrieve information about a specific country. The [countries]({{page.baseurl}}/graphql/queries/directory-countries.html) query is used to retrieve a list of countries available in the system.
```

File: src/guides/v2.3/graphql/queries/directory-country.md [hunk 5]
```diff
@@ -5,15 +5,15 @@ redirect_from:
...
 
 ## Syntax
 
 `{country(id: String) {Country}}`
 
 ## Example usage
 
-The following query returns information about Australia.
+The following query returns information about Australia. You must pass `id` as a two-letter abbreviation of the country to retrieve country details.
```
Comment (hguthrie) [inline src/guides/v2.3/graphql/queries/directory-country.md:16]:
```suggestion
The following query uses a two-letter abbreviation for the country ID (id: "AU"), which returns information about Australia.
```

**Review by hguthrie — CHANGES_REQUESTED** [review]
Thanks for the contribution. Here are some grammatical fixes.

**Comment by RakeshJesadiya** [comment]
@hguthrie @keharper I have updated the changes based on your suggestion. Thanks.

**Review by keharper — APPROVED** [review]

**Pull request merged**